app/models/concerns/host_searching.rb

```diff
@@ -5,7 +5,7 @@ module HostSearching
   extend ActiveSupport::Concern
 
   included do
-    scoped_search on: %i[display_name os_major_version os_minor_version]
+    scoped_search on: %i[id display_name]
```

Comment:
We have frontend tasks that will require these search fields. Here's one way to query them with ActiveRecord:

```rb
Host.where('system_profile @> ?', {operating_system: {major: 7}}.to_json)
```